### PR TITLE
Filter out dropped columns

### PR DESCRIPTION
**Pull request by thecatontheflat**
https://github.com/dimitri/pgcopydb/issues/275

### CODE REVIEW

File: src/bin/pgcopydb/schema.c [hunk 1]
```diff
@@ -749,7 +749,7 @@ struct FilteringQueries listSourceTablesSQL[] = {
 		"                                on i.indrelid = a.attrelid "
 		"                               and a.attnum = ANY(i.indkey) "
 		"                               and i.indisprimary "
-		"                   where a.attrelid = c.oid "
+		"                   where a.attrelid = c.oid and a.attisdropped = false"
```
Comment (dimitri) [inline src/bin/pgcopydb/schema.c:752]: I think I would prefer the notation `and not a.attisdropped`. Could you update the patch?

**Review by dimitri — CHANGES_REQUESTED** [review]
Nitpicking, see details.

File: src/bin/pgcopydb/schema.c [hunk 2]
```diff
@@ -824,7 +824,7 @@ struct FilteringQueries listSourceTablesSQL[] = {
...
```
Comment (thecatontheflat) [inline src/bin/pgcopydb/schema.c:827]:
```suggestion
		"                   where a.attrelid = c.oid and not a.attisdropped "
```

File: src/bin/pgcopydb/schema.c (hunk 1 above)
Comment (thecatontheflat) [inline src/bin/pgcopydb/schema.c:752]:
```suggestion
		"                   where a.attrelid = c.oid and not a.attisdropped "
```

File: src/bin/pgcopydb/schema.c [hunk 3]
```diff
@@ -901,7 +901,7 @@ struct FilteringQueries listSourceTablesSQL[] = {
...
```
Comment (thecatontheflat) [inline src/bin/pgcopydb/schema.c:904]:
```suggestion
		"                   where a.attrelid = c.oid and not a.attisdropped "
```

File: src/bin/pgcopydb/schema.c [hunk 4]
```diff
@@ -992,7 +992,7 @@ struct FilteringQueries listSourceTablesSQL[] = {
...
```
Comment (thecatontheflat) [inline src/bin/pgcopydb/schema.c:995]:
```suggestion
		"                   where a.attrelid = c.oid and not a.attisdropped "
```

File: src/bin/pgcopydb/schema.c [hunk 5]
```diff
@@ -1072,7 +1072,7 @@ struct FilteringQueries listSourceTablesSQL[] = {
...
```
Comment (thecatontheflat) [inline src/bin/pgcopydb/schema.c:1075]:
```suggestion
		"                   where a.attrelid = c.oid and not a.attisdropped "
```

File: src/bin/pgcopydb/schema.c (hunk 1 above)
Comment (thecatontheflat) [reply src/bin/pgcopydb/schema.c]: Done

**Review by dimitri — APPROVED** [review]

**Comment by dimitri** [comment]
Thanks for your contribution @thecatontheflat !

**Pull request merged**